releases/axelard/2025-11-v1.3.2.md

```diff
@@ -310,66 +322,33 @@ EOF
 
-8. If your key has disappeared from the keyring file.
-
-- `No records were found in keyring`
-  Then please re-add the broadcaster account by running `axelard keys add broadcaster --recover --keyring-backend file`
-
-9. If you hit an error saying: `failed to execute message: too many heartbeats received from operator in vald`. Then ensure that you have `indexer="kv"` in your `config.toml` file under the `[tx_index]` key
+8. If you hit an error saying: `failed to execute message: too many heartbeats received from operator in vald`. This just requires waiting for your node to catch up. We recommend starting vald only once the node has fully caught up and is producing blocks
 
 Upon the completion of this testnet migration several key findings were observed:
 
 - **Archival node:** Migration took **~12.5 hours**, plus additional catch-up time.
-    - Peak memory usage observed: **~42 GB**
+    - Peak memory usage observed: **~64 GB**
 - **Validator node:** Migration took **~1 hour**, plus catch-up.
-    - Peak memory usage observed: **~40 GB**
+    - Peak memory usage observed: **~64 GB**
 - Other RPC operators reported migration durations ranging **from ~50 to ~100 minutes**.
 
-#### Additional Test Results
-
-Follow-up testing on mainnet RPC provided more detailed resource utilization metrics:
-
-**Archival RPC Nodes:**
-
-- CPU: **~3 cores** (with spikes, though CPU throttling will occur)
-- Memory: **~42 GB** short peak (with spikes up to **~45 GB** for very brief periods)
-
-**Standard RPC Nodes:**
-
-- CPU: **~2.5 cores** (with spikes, though CPU throttling will occur)
-- Memory: **~38 GB** average peak
-    - Individual nodes varied: some peaked at **~40 GB**, while others required no more than **~32 GB**
-
-#### Migration Behavior Observed
-
-- If the `axelar-core` process restarted during migration, the migration started over from the beginning, rather than resuming from prior progress.
-- During the migration, some nodes were restarted due to running out of memory, as migration memory usage peaked at **2-3x normal usage (~40–42 GB)**.
 
 #### Action Items for Validators / RPC Operators
 
-- **Be aware that migration memory usage may reach ~40–42 GB.**  
+- **Be aware that migration memory usage may reach ~64 GB.**  
   If running in environments with memory limits (e.g., Kubernetes), ensure your limits are configured appropriately to avoid OOM-kills during the migration phase.
 
 - **Avoid restarts during the migration phase**, as any restart will cause the migration to begin again and significantly extend total upgrade time.
 
 - **Note:** After all modules finish upgrading, block production will resume once quorum of validators by stake have completed the upgrade. Depending on validator response rate, this can add an additional **30–90 minutes** to the overall migration duration.
```

Comment:
instead of having the suggestions hidden at the bottom, add the notes in the relevant deployment steps (but make it very concise)